CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
   ## v6.15.0
 
+  * **Official Ruby 3.0 support**
+    
+    The ruby agent has been verified to run on ruby 3.0.0
+
+  * **Added support for Rails 6.1**
+
+    The ruby agent has been verified to run with Rails 6.1
+
```

Comment:
Please call out our contributor with special thanks here!

Reply:
Great point! thank you for the reminder!